docs/ovms_quickstart.md

```diff
@@ -101,7 +101,7 @@ For more information:
 2. Put the image in a folder by itself. The script runs inference on all images in the folder.
 
 ```
-curl --create-dirs https://raw.githubusercontent.com/openvinotoolkit/model_server/main/demos/common/static/images/people/people1.jpeg -o images/people1.jpeg
+curl --create-dirs https://raw.githubusercontent.com/openvinotoolkit/model_server/develop/demos/common/static/images/people/people1.jpeg -o images/people1.jpeg
```

Comment:
do not change the branch do develop, it will be merged to main soon

Reply:
reverted